styles.css

```diff
@@ -20,8 +17,49 @@ p {
   align-items: center;
 }
 .menu__link {
-  padding: 0 13px;
+  position: relative;
+  margin-right: 26px;
+  font-family: 500 18px/21px 'Roboto', sans-serif, arial, helvetica;
+  text-decoration: none;
+  color: #000;
+  transition: opacity .5s;
 }
 .menu__link:last-child {
-  padding-right: 0;
+  margin-right: 0;
+}
+.menu__link_active::after {
+  position: absolute;
+  display: block;
+  content:'';
+  left: 0;
+  width: 100%;
+  height: 3px;
+  background: #0056EC;
+}
+.menu__link:hover {
+  opacity: .5;
+}
+.menu__link:hover::after {
+  position: absolute;
+  display: block;
+  content: '';
+  left: 0;
+  width: 100%;
+  height: 3px;
+  background: #0056EC;
+}
+.menu__link::after {
```

Comment:
Codacy found an issue: [Expected selector ".menu__link::after" to come before selector ".menu__link:hover::after" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/html-css-2-0/pullRequest?prid=6269228)